hbase-client/src/main/java/org/apache/hadoop/hbase/client/ZKAsyncRegistry.java

```diff
@@ -229,6 +244,44 @@ private void getMetaRegionLocation(CompletableFuture<RegionLocations> future,
         });
   }
 
+  @Override
+  public CompletableFuture<TableState> getMetaTableState() {
```

Comment:
think you missed the tests for this? we can add to TestZKAsyncRegistry.

Reply:
Yes. Let me add. Thanks.

Reply:
Added small test in next push. It was interesting since TestZKRegistry has replicas on meta itself which made things more interesting.